### DISCUSSION TITLE
Mixed html and ppt slides

### DISCUSSION THREAD

**Discussion by jwang-lilly** [quarto-dev/quarto-cli#2809] [Q&A]
Hi,

I have a large analysis in qmd file using HTML as output type. However, I must prepare ppt slides to share with some folks who don't need all those details in the HTML file.  

Besides starting from scratch inserting figures saved from HTML run to prepare ppt, what other ways are available to approach this?  Could we designate slides within the .qmd file meant for HTML output for selected key content in a ppt? 

Thanks much for any advice.

**Comment by rgaiacs** [comment]
Hi @JianWang2016, thanks for the question.

Did you try [conditional content](https://quarto.org/docs/authoring/conditional.html#format-matching)

The follow works for me.

``````
---
title: "Minimal Working Example"
format: 
  html: default
  pptx: default
---

## Introduction

::: {.content-visible when-format="html"}

## Large Analysis

```{r}
1 + 1
```

:::

## Awesome Slides

:-D
``````

[Above Quarto document with output files.](https://github.com/quarto-dev/quarto-cli/files/9760962/mwe.zip)
